<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <meta name="viewport" content="width=device-width">

        <title>MISO : 5. Type Data</title>
        <meta name="description" content="An open-source LIMS for NGS sequencing centres">

        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
        <link rel="stylesheet" href="/miso-lims/0.2.x/css/syntax.css">
        <link rel="stylesheet" href="/miso-lims/0.2.x/css/main.css">
				<link rel="icon" href="/miso-lims/0.2.x/favicon.ico" type="image/x-icon">
    </head>
    <body>
<a href="https://github.com/miso-lims/miso-lims"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png"></a>
        <div class="container">
            <div class="row">
                <div id="header" class="col-sm-12">
                    <h2><a class="brand" href="/miso-lims/0.2.x/"><img src="/miso-lims/0.2.x/images/MISO.png" alt="MISO's logo, a bowl of miso soup"/>MISO</a>
    <small>An open-source LIMS for NGS sequencing centres</small>
</h2>

                </div>
            </div>

            <div class="row">
                
                
                    <div id="navigation" class="col-sm-2">
                        <ul class="nav nav-list">
    <!--<li><a href="/miso-lims/0.2.x/">Home</a></li>-->
    
        
        

        
    
        
        

        
            
                <li class="nav-header">User Guides</li>
            
            <li data-order="1"><a href="/miso-lims/0.2.x/usr/user-manual-table-of-contents.html">User Manual</a></li>
        
    
        
        

        
            
                <li class="nav-header">Admin Guides</li>
            
            <li data-order="2"><a href="/miso-lims/0.2.x/adm/baremetal-installation-guide.html">Installing MISO on baremetal</a></li>
        
            
            <li data-order="2"><a href="/miso-lims/0.2.x/adm/compose-installation-guide.html">Installing MISO with docker-compose</a></li>
        
            
            <li data-order="2"><a href="/miso-lims/0.2.x/adm/admin-guide.html">Administrator's Manual</a></li>
        
            
            <li data-order="1"><a href="/miso-lims/0.2.x/adm/installation-guide.html">Building and Deploying</a></li>
        
    
        
        

        
            
                <li class="nav-header">Developer Guides</li>
            
            <li data-order="2"><a href="/miso-lims/0.2.x/dev/rest-testing.html">REST Testing</a></li>
        
            
            <li data-order="2"><a href="/miso-lims/0.2.x/dev/maven-artifacts.html">Maven artifacts</a></li>
        
            
            <li data-order="2"><a href="/miso-lims/0.2.x/dev/eclipse-code-formatting.html">Eclipse Code Formatting</a></li>
        
            
            <li data-order="2"><a href="/miso-lims/0.2.x/dev/code-style.html">Code Style</a></li>
        
            
            <li data-order="1"><a href="/miso-lims/0.2.x/dev/developers-manual.html">Developer's Manual</a></li>
        
    
        
        

        
    
        
        

        
    
<!-- List additional links. It is recommended to add a divider
    e.g. <li class="divider"></li> first to break up the content. -->
</ul>

                    </div>

                    <div id="content" class="col-sm-10">
                        <div class="page-header">
    <h2>5. Type Data
        
    </h2>
</div>

<p>Type data describes sets of items listed as dropdown options in various forms throughout MISO. Most of these options
can be modified to suit your institute. In most cases, only MISO administrators can make these changes.</p>

<p>Each type that can be modified within MISO has a list page. You can find most of these by clicking the appropriate link
in the Misc list in the navigation menu. Types that are only used in detailed sample mode appear in the Institute
Defaults list instead.</p>

<p>To add new items, click the “Add” button in the toolbar at the top of the list. In the dialog, enter the quantity that
you would like to create, and click the “Create” button. This will take you to the bulk Create page. Enter the
information for the new items, and click the “Save” button at the top right to confirm.</p>

<p>To edit existing items, go to the list page, check the checkboxes beside the items you would like to edit, and click
the “Edit” button in the toolbar at the top of the table. This will bring you to the bulk Edit page, which is identical
to the bulk Create page mentioned above. Here, you can modify the existing items. Be sure to click the “Save” button at
the top right to confirm any changes. Keep in mind that any changes you make will affect all existing items that are
already using the affected options. For example, if you rename a kit descriptor, existing libraries that are already
using that kit will be affected.</p>

<p>To delete items, go to the list page, check the checkboxes beside the items you would like to delete, and click the
“Delete” button in the toolbar at the top of the table. You will only be able to delete items that are not in use.</p>

<p><a name="reference_genomes"></a></p>
<h2 id="51-reference-genomes">5.1. Reference Genomes</h2>

<p>Reference genome refers to a file that sequenced data will be aligned to. This is set at the project level, and can be
changed on both the Create Project and Edit Project page. A reference genome may specify a default scientific name to
be used for new samples within the project.</p>

<p>MISO administrators can add, edit, and delete reference genomes using the standard interface. A reference genome can
only be deleted if it is not used by any existing projects.</p>

<p><a href="#">Return to top</a></p>

<p><a name="sample_types"></a></p>
<h2 id="52-sample-types">5.2. Sample Types</h2>

<p><strong>WARNING</strong>: Sample type is defined by <a href="https://www.ncbi.nlm.nih.gov/">NCBI</a> and is required in order to submit samples
to the ENA (see the <a href="european_nucleotide_archive_support.html">ENA section</a>). The values available in MISO should be
limited to the default options, which are the same as defined in the ENA schema. If these values are modified, or
additional options are added, your data may not be valid for ENA submission.</p>

<p>It is possible to archive sample types if they are not used within your organization. An archived sample type will not
show up in the sample type options when creating or editing samples, unless the samples being edited have already used
that sample type. Sample types can always be unarchived if they are needed in the future.</p>

<p>MISO administrators can add, edit, and delete sample types using the standard interface. Sample types can be set as
archived on the bulk Create or Edit page. A sample type can only be deleted if the option has not been used by any
existing samples.</p>

<p><a href="#">Return to top</a></p>

<p><a name="sample_classes_and_categories"></a></p>
<h2 id="53-sample-classes-and-categories">5.3. Sample Classes and Categories</h2>

<p>Note: These items only apply if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>Detailed sample mode introduces a hierarchy of samples. Sample classes identify where a sample fits into the hierarchy.
Sample categories are used to group sample classes into levels. Sample categories may not be modified. See
<a href="site_configuration.html#detailed_sample_mode">Detailed Sample Mode</a> for more information about the sample categories
and ghost samples.</p>

<p>Sample classes, and the relationships between them define the sample hierarchy. Both the classes and relationships can
be customized to suit your requirements. Figure 1 shows the sample hierarchy used at OICR. Sample
relationships have a parent and child class. This means that if you have a sample of the parent class, you can
propagate a sample of the child class. In figure 1, the relationships are illustrated using arrows that point
from the parent class to the child class.</p>

<p><img class="userman-figure" src="/miso-lims/0.2.x/images/type-data-sample-hierarchy.png" alt="OICR sample hierarchy" /></p>

<p><em>figure 1: OICR sample hierarchy</em></p>

<p>Sample classes and valid relationships can only be modified via direct access to the MISO database, and so must be done
by a MISO administrator. A custom hierarchy must follow these rules:</p>

<ul>
  <li>there may be only one identity class</li>
  <li>with the exception of identity classes, any class may be parented to itself</li>
  <li>any class may be parented to another class in the same category</li>
  <li>an aliquot class must always be parented to a single stock class</li>
  <li>a stock class may be parented to a tissue processing class, but must always be parented to a single tissue class</li>
  <li>a tissue class must be parented to the identity class</li>
</ul>

<p>Sample classes may be archived, which means that they will no longer available when creating new samples. Similarly,
relationships between classes may also be archived. This means that you will no longer be able to propagate samples
to create that type of relationship. Using the OICR sample hierarchy in figure 1 as an example, we could
archive the relationship between Slide and LCM Tube to prevent users from propagating LCM Tubes from Slide samples.</p>

<p>Another option is to prevent direct creation of a sample class. This means that the sample class will be available as
an option for the aliquot class to use when receiving libraries, but it won’t be available as an option when creating
or propagating samples. This is useful if you want to have an “Unknown” type of sample class for cases where libraries
are received and not much is known about the samples they were made from.</p>

<p><a href="#">Return to top</a></p>

<p><a name="tissue_materials"></a></p>
<h2 id="54-tissue-materials">5.4. Tissue Materials</h2>

<p>Note: This item only applies if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>Tissue materials describe how the tissue was prepared and may include options such as ‘Fresh Frozen,’ ‘FFPE,’ and ‘Blood.’
Tissue material is an optional field for tissue samples.</p>

<p>Any user can add tissue materials using the standard inerface. MISO administrators can also edit and delete tissue
materials. A tissue material can only be deleted if the option has not been used by any existing samples.</p>

<p><a href="#">Return to top</a></p>

<p><a name="tissue_origins"></a></p>
<h2 id="55-tissue-origins">5.5. Tissue Origins</h2>

<p>Note: This item only applies if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>Tissue origins describe what part of the donor or organism the sample was taken from. For human donors, options may
include ‘Brain,’ ‘Lung,’ and ‘Pancreas.’</p>

<p>MISO administrators can add, edit, and delete tissue origins using the standard interface. A tissue origin can only be
deleted if the option has not been used by any existing samples.</p>

<p><a href="#">Return to top</a></p>

<p><a name="tissue_types"></a></p>
<h2 id="56-tissue-types">5.6. Tissue Types</h2>

<p>Note: This item only applies if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>Tissue Type is a classifier for tissue samples. It may be used to categorize the sample. For example, tissue types at
OICR include the following (and others).</p>

<ul>
  <li>Reference Tissue: Reference or non-tumour, non-diseased tissue sample. Typically used as a donor-specific comparison
to a diseased tissue, usually a cancer</li>
  <li>Primary Tumor Tissue</li>
  <li>Metastatic Tumor Tissue</li>
  <li>Xenograft Tissue: Xenograft derived from some tumour</li>
  <li>Organoid</li>
</ul>

<p>MISO administrators can add, edit, and delete tissue types using the standard interface. A tissue type can only be
deleted if the option has not been used by any existing samples.</p>

<p><a href="#">Return to top</a></p>

<p><a name="sample_purposes"></a></p>
<h2 id="57-sample-purposes">5.7. Sample Purposes</h2>

<p>Note: This item only applies if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>Sample purpose describes the reason an aliquot was created. Example purposes include, ‘Library,’ ‘Validation,’ and
‘Extra.’</p>

<p>MISO administrators can add, edit, and delete sample purposes using the standard interface. A sample purpose can only
be deleted if the option has not been used by any existing samples.</p>

<p><a href="#">Return to top</a></p>

<p><a name="labs_and_institutes"></a></p>
<h2 id="58-labs-and-institutes">5.8. Labs and Institutes</h2>

<p>Note: These items only apply if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>If a sample was received from an external lab, this can be recorded at the tissue level. Each lab belongs to an
institute.</p>

<p>Any user may create new labs and institutes in MISO using the standard interface. Administrators may also
edit and delete existing labs and institutes. You will only be allowed to delete a lab if it has not been used by any
existing samples. You will only be allowed to delete an institute if it has no labs associated with it. This means that
if you want to delete an institute, you will first have to delete all of its labs.</p>

<p><a href="#">Return to top</a></p>

<p><a name="detailed_qc_status"></a></p>
<h2 id="59-detailed-qc-status">5.9. Detailed QC Status</h2>

<p>Note: This item only applies if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>In plain sample mode, samples have a “QC Passed?” field that can be set to true (passed), false (failed), or unknown.
Detailed sample mode expands upon this with detailed QC status. A detailed QC status does specify a QC status of
passed, failed, or unknown, but also provides more specific information about the reason. Here are some examples:</p>

<ul>
  <li>“Okd by Collaborator” (QC status: passed) - may indicate that a collaborator has approved use of the sample despite
other QC checks</li>
  <li>“Waiting: Receive Tissue” (QC status: unknown) - indicates that the tissue has not yet been received, so no QC has
been performed yet</li>
  <li>“Refused Consent” (QC status: failed) - indicates that while the sample may have passed QC, the donor has revoked
consent for it to be used</li>
</ul>

<p>MISO administrators can add, edit, and delete detailed QC statuses using the standard interface. A detailed QC status
can only be deleted if the status has not been used by any existing samples.</p>

<p><a href="#">Return to top</a></p>

<p><a name="stains"></a></p>
<h2 id="510-stains">5.10. Stains</h2>

<p>Note: This item only applies if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>Different stains such as Cresyl Violet and Hematoxylin+Eosin may be used on a slide sample. These stains can be broken
into categories, which is useful if you use many different stains.</p>

<p>MISO administrators can add, edit, and delete both stains and stain categories using the standard interface. A stain
can only be deleted if the option has not been used by any existing samples. A stain category can only be deleted if
there are no stains associated with it.</p>

<p><a href="#">Return to top</a></p>

<p><a name="library_types"></a></p>
<h2 id="511-library-types">5.11. Library Types</h2>

<p>Library type is a classifier for libraries that depends on the platform type. Library types for Illumina libraries
include ‘Paired End,’ ‘Single End,’ and ‘Mate Pair,’ for example. Depending on your naming scheme, the library type
abbreviation may be included in library aliases. Library types may be archived so that they are no longer available for
new libraries.</p>

<p>MISO administrators can add, edit, and delete library types using the standard interface. The platform a library type
is linked to cannot be changed if the library type has been used for any existing libraries or library templates. A
library type can only be deleted if the status has not been used by any existing libraries or library templates.</p>

<p><a href="#">Return to top</a></p>

<p><a name="library_designs"></a></p>
<h2 id="512-library-designs">5.12. Library Designs</h2>

<p>Note: This item only applies if your site uses <a href="site_configuration.html#detailed_sample_mode">detailed sample mode</a>.</p>

<p>A library design is a grouping of <a href="#library_selection_types">selection</a> and <a href="#library_strategy_types">strategy</a> types
and a design code. The library designs that are available for a library depend on the sample class of the aliquot that
the library was propagated from.</p>

<p>MISO administrators can add, edit, and delete both library designs and library design codes using the standard
interface. Most attributes of a library design cannot be modified if the design has been used by any existing
libraries, as this would cause the libraries’ attributes to be out of sync. A library design can only be deleted if it
has not been used by any existing libraries. A library design code can only be deleted if it has not been used by any
existing libraries or library designs.</p>

<p><a href="#">Return to top</a></p>

<p><a name="library_selection_types"></a></p>
<h2 id="513-library-selection-types">5.13. Library Selection Types</h2>

<p><strong>WARNING</strong>: Library selection type is defined by <a href="https://www.ncbi.nlm.nih.gov/">NCBI</a> as the “method used to enrich
the target in the sequence library preparation” and is required in order to submit libraries to the ENA (see the
<a href="european_nucleotide_archive_support.html">ENA section</a>). The values available in MISO should be limited to the default
options, which are the same as defined in the ENA schema. If these values are modified, or additional options are
added, your data may not be valid for ENA submission.</p>

<p>MISO administrators can add, edit, and delete library selection types using the standard interface. A library selection
type can only be deleted if it has not been used by any existing libraries.</p>

<p><a href="#">Return to top</a></p>

<p><a name="library_strategy_types"></a></p>
<h2 id="514-library-strategy-types">5.14. Library Strategy Types</h2>

<p><strong>WARNING</strong>: Library strategy type is defined by <a href="https://www.ncbi.nlm.nih.gov/">NCBI</a> as the “sequencing type intended
for this library” and is required in order to submit libraries to the ENA (see the
<a href="european_nucleotide_archive_support.html">ENA section</a>). The values available in MISO should be limited to the default
options, which are the same as defined in the ENA schema. If these values are modified, or additional options are
added, your data may not be valid for ENA submission.</p>

<p>MISO administrators can add, edit, and delete library strategy types using the standard interface. A library strategy
type can only be deleted if it has not been used by any existing libraries.</p>

<p><a href="#">Return to top</a></p>

<p><a name="kit_descriptors"></a></p>
<h2 id="515-kit-descriptors">5.15. Kit Descriptors</h2>

<p>A kit descriptor describes a kit product that is used for library or run preparation. The types of kits available in
MISO are:</p>

<ul>
  <li>Library kit - used for libraries</li>
  <li>Sequencing kit - not currently used in MISO</li>
  <li>Clustering kit - used for sequencing containers</li>
  <li>Multiplexing kit - used for sequencing containers</li>
  <li>Extraction kit - not currently used in MISO</li>
</ul>

<p>The Kits lost page is broken into tabs for the different kit types.</p>

<p>To add a new kit descriptor, click the “Add” button in the toolbar at the top of the table on the Kits list page. This
will take you to the Create Kit Descriptor page. Enter all of the kit details, then click the “Save” button at the top
right to create the kit descriptor.</p>

<p>Once a kit has been created, it is possible to link existing <a href="#targeted_sequencing">Targeted Sequencing</a> values to the
kit descriptor. Then, if a library is prepared using this kit descriptor, aliquots created from the library can be 
assigned a targeted sequencing from the targeted sequencings linked to the kit descriptor. This means that if a desired
targeted sequencing is not available for a library aliquot, this can be solved by linking the targeted sequencing to
the kit descriptor of the library aliquot’s parent library.</p>

<p><a href="#">Return to top</a></p>

<p><a name="indices"></a></p>
<h2 id="516-indices">5.16. Indices</h2>

<p>Indices, also known as barcodes or primers, are sequences that are added to libraries in order to identify which
library each individual DNA fragment came from in a multiplexed pool. An index family is a collection of these indices
that are intended to be used together. Index sequences should all be unique within an index family.</p>

<p>Some index families are dual-indexed. This means that one or two indices can be added to each library. This increases
the possibilities for unique sequences. Some dual-indexed families have an index 1 and index 2 that are always matched
together. These are referred to as “unique dual index” families within MISO. In other index families, any index 1 may
be matched with any index 2.</p>

<p>Some Illumina sequencers, including the NextSeq, sequence the index 2 as the
reverse complement of other sequencers. MISO has an option to set the
sequencing platform to use the normal index in the interface and provide the
reverse complement in sample sheets and though the Pinery data export
interface.</p>

<p>Index families may be archived if they are no longer needed. This will prevent them from showing up in the options when
creating new libraries.</p>

<p>Indices can be viewed on the Indices list page. To get there, click “Indices” in the Misc list in the menu on the
left side of the screen. The page is broken into tabs for each platform type. Modifications to indices and index
families can only be made via direct access to the MISO database. As such, they must be performed by a MISO
adminstrator.</p>

<p><a href="#">Return to top</a></p>

<p><a name="library_spike-ins"></a></p>
<h2 id="517-library-spike-ins">5.17. Library Spike-Ins</h2>

<p>A library spike-in is a control added to a library for normalization. MISO administrators can add, edit, and delete
library spike-ins using the standard interface. A library spike-in can only be deleted if it has not been used by any
existing libraries.</p>

<p><a href="#">Return to top</a></p>

<p><a name="targeted_sequencing"></a></p>
<h2 id="518-targeted-sequencing">5.18. Targeted Sequencing</h2>

<p>Targeted sequencing allows specific sections of the genome to be sequenced resulting in more depth and lower cost. The
targeted sequencing value specified on a library aliquot should be associated with a .bed file that will be used for
analysis and QC, though this file is not tracked within MISO.</p>

<p>Targeted sequencing values may be archived if they are no longer needed. This will prevent them from showing up in the
options when creating new library aliquots.</p>

<p>Modifications to targeted sequencing values can only be made via direct access to the MISO database. As such, they must
be performed by a MISO adminstrator.</p>

<p><a href="#">Return to top</a></p>

<p><a name="order_purposes"></a></p>
<h2 id="519-order-purposes">5.19. Order Purposes</h2>

<p>Order purposes describe the reason for a sequencing order, such as QC or production. A pool order may also specify an
order purpose if it includes sequencing requirements.</p>

<p>MISO administrators can add, edit, and delete order purposes using the standard interface. An order purpose can only be
deleted if it has not been used by any existing pool orders or sequencing orders.</p>

<p><a href="#">Return to top</a></p>

<p><a name="qc_types"></a></p>
<h2 id="520-qc-types">5.20. QC Types</h2>

<p>QC types identify instruments or methods used to QC an item. A QC type may target samples, libraries, pools, sequencing
containers, or runs. Some QC types may be linked to one of the target’s fields. For example, a “Volume Check” QC type
might be linked to a sample’s volume. These QCs can be set to auto-update the field, so that when a new QC is entered,
the item’s related field is updated automatically.</p>

<p>MISO administrators can add and edit QC types using the standard interface.</p>

<p><a href="#">Return to top</a></p>

<p><a name="partition_qc_types"></a></p>
<h2 id="521-partition-qc-types">5.21. Partition QC Types</h2>

<p>Partition QCs describe the status of a sequenced partition and allows individual partitions within a run to have their
own status, rather than grouping them all together with the run’s status. Partition QCs can be used to determine
whether the partition counts towards order fulfillment, and whether downstream analysis should be run on the partition.</p>

<p>MISO administrators can add, edit, and delete partition QC types using the standard interface. A partition QC type can
only be deleted if the type has not been used by any existing partitions.</p>

<p><a href="#">Return to top</a></p>

<p><a name="box_sizes"></a></p>
<h2 id="522-box-sizes">5.22. Box Sizes</h2>

<p>A box in MISO describes a physical container that holds multiple samples, libraries, library aliquots, and/or pools.
Box sizes define the possible dimensions of these boxes. A box size may be marked as scannable. This means that the box
can be scanned using a bulk barcode scanner (see <a href="site_configuration.html#barcode_scanners">Barcode Scanners</a>).</p>

<p>MISO administrators can add, edit, and delete box sizes using the standard interface. Box size dimensions cannot be
modified if there are any existing boxes using the size. A box size can only be deleted if the size has not been used
by any existing boxes.</p>

<p><a href="#">Return to top</a></p>

<p><a name="box_uses"></a></p>
<h2 id="523-box-uses">5.23. Box Uses</h2>

<p>A box in MISO describes a physical container that holds multiple samples, libraries, library aliquots, and/or pools.
Box uses are used to categorize these boxes. The Boxes list page is broken into tabs - one for each box use.</p>

<p>MISO administrators can add, edit, and delete box uses using the standard interface. A box use can only be deleted if
it has not been used by any existing boxes.</p>

<p><a href="#">Return to top</a></p>

<p><a name="instrument_models"></a></p>
<h2 id="524-instrument-models">5.24. Instrument Models</h2>

<p>Instrument models describe the instrument types that can be registered in MISO. These can be broken into three
categories:</p>

<ul>
  <li>Sequencers: Used for sequencer runs</li>
  <li>Array scanners: Used for array runs</li>
  <li>Others: Not used within MISO; however, they may be registered in order to track service records or for other purposes</li>
</ul>

<p>Modifications to instrument models can only be made via direct access to the MISO database. As such, they must be
performed by a MISO adminstrator.</p>

<p><a href="#">Return to top</a></p>

<p><a name="sequencing_container_models"></a></p>
<h2 id="525-sequencing-container-models">5.25. Sequencing Container Models</h2>

<p>Most instrument models may be run with different sequencing container models. Some sequencing container models may also
be used by multiple instrument models. A sequencing container must specify its model, and this is used to determine
which runs the container can be attached to. Modifications to sequencing container models can only be made via direct
access to the MISO database. As such, they must be performed by a MISO adminstrator.</p>

<p><a href="#">Return to top</a></p>

<p><a name="sequencing_parameters"></a></p>
<h2 id="526-sequencing-parameters">5.26. Sequencing Parameters</h2>

<p>Sequencing parameters describe the settings that a sequencer run was configured with for a run. They include details
such as the chemistry version, read length, and whether the run was paired end. When making orders, you can choose the
parameters that are required. Later, MISO will look at the sequencing parameters used by a sequencing run in order to
determine whether the order has been completed.</p>

<p>Modifications to sequencing parameter options can only be made via direct access to the MISO database. As such, they
must be performed by a MISO adminstrator.</p>

<p><a href="#">Return to top</a></p>

<p><a name="oxford_nanopore_flow_cell_pore_versions"></a></p>
<h2 id="527-oxford-nanopore-flow-cell-pore-versions">5.27. Oxford Nanopore Flow Cell Pore Versions</h2>

<p>Oxford Nanopore flow cells (sequencing containers) specify a pore version. This can be recorded in MISO for any Oxford
Nanopore flow cells. Modifications to pore version options can only be made via direct access to the MISO database. As
such, they must be performed by a MISO adminstrator.</p>

<p><a href="#">Return to top</a></p>

<p><a name="array_models"></a></p>
<h2 id="528-array-models">5.28. Array Models</h2>

<p>Array models are similar to sequencing container models, except they describe the chip or cartridge that the array is
loaded into, including the number of samples it can hold.</p>

<p>MISO administrators can add, edit, and delete array models using the standard interface. An array model can only be
deleted if it is not used by any existing arrays.</p>

<p><a href="#">Return to top</a></p>

<p><a name="study_types"></a></p>
<h2 id="529-study-types">5.29. Study Types</h2>

<p><strong>WARNING</strong>: Study type is defined by <a href="https://www.ncbi.nlm.nih.gov/">NCBI</a> as “expressing the overall purpose of the
study.” This property is required in order to submit studies to the ENA (see the
<a href="european_nucleotide_archive_support.html">ENA section</a>). The values available in MISO should be limited to the default
options, which are the same as defined in the ENA schema. If these values are modified, or additional options are
added, your data may not be valid for ENA submission.</p>

<p>MISO administrators can add, edit, and delete study types using the standard interface. A study type can only be
deleted if it is not used by any existing studies.</p>

<p><a href="#">Return to top</a></p>



                    </div>
                
            </div>

            

            <div class="row">
                <div id="footer" class="col-sm-12">
                    Documentation for <a href="https://github.com/TGAC/miso-lims">MISO</a>

                </div>
            </div>
        </div>

        <script>
            function orderNav() {
                var list,
                    section,
                    header,
                    sections = [],
                    lists = {},
                    headers = {};

                var navUl = document.querySelectorAll('#navigation ul')[0],
                    navLis = document.querySelectorAll('#navigation ul li');

                if (!navUl) return;

                for (var i = 0; i < navLis.length; i++) {
                    var order, li = navLis[i];

                    if (li.classList.contains('nav-header')) {
                        section = li.textContent || li.innerText;
                        sections.push(section);
                        headers[section] = li;
                        continue;
                    }

                    if (!lists[section]) {
                        lists[section] = [];
                    }

                    order = parseFloat(li.getAttribute('data-order'))
                    lists[section].push([order, li]);
                }

                for (var i = 0; i < sections.length; i++) {
                    section = sections[i];
                    list = lists[section].sort(function(a, b) {
                        return a[0] - b[0];
                    });

                    if (header = headers[section]) {
                        navUl.appendChild(header);
                    }
                    for (var j = 0; j < list.length; j++) {
                        navUl.appendChild(list[j][1]);
                    }
                }
            }

            if (document.querySelectorAll) orderNav();
        </script>
        
    </body>
</html>
